cmd/expiration-mailer/main.go

```diff
@@ -286,9 +289,31 @@ func (m *mailer) findExpiringCertificates() error {
 			)
 		}
 		if err != nil {
-			m.log.AuditErr(fmt.Sprintf("expiration-mailer: Error loading certificates: %s", err))
-			return err // fatal
+			m.log.AuditErr(fmt.Sprintf("expiration-mailer: Error loading certificate statuses: %s", err))
+			return err
+		}
+
+		// Now we can sequentially retrieve the certificate details for each of the
+		// certificate status rows
+		var certs []core.Certificate
+		for _, cs := range statuses {
```

Comment:
It's only faster in the case the migration is applied - I thought it was clearer to write this so that both branches were finding certificate statuses and then certificates as opposed to one branch that did both at once.